ospf6d/ospf6_neighbor.c

```diff
@@ -1086,6 +1103,39 @@ DEFUN (show_ipv6_ospf6_neighbor_one,
 				json, JSON_C_TO_STRING_PRETTY));
 		json_object_free(json);
 	}
+
+	return CMD_SUCCESS;
+}
+
+DEFUN(show_ipv6_ospf6_neighbor_one, show_ipv6_ospf6_neighbor_one_cmd,
+      "show ipv6 ospf6 [vrf <NAME|all>] neighbor A.B.C.D [json]",
+      SHOW_STR IP6_STR OSPF6_STR VRF_CMD_HELP_STR
+      "All VRFs\n"
+      "Neighbor list\n"
+      "Specify Router-ID as IPv4 address notation\n" JSON_STR)
+{
+	int idx_ipv4 = 4;
+	struct ospf6 *ospf6;
+	struct listnode *node;
+	const char *vrf_name = VRF_DEFAULT_NAME;
+	bool all_vrf = false;
+	int idx_vrf = 0;
+
+	OSPF6_FIND_VRF_ARGS(argv, argc, idx_vrf, vrf_name, all_vrf);
+	if (idx_vrf > 0)
+		idx_ipv4 += 2;
+	if (all_vrf) {
+		for (ALL_LIST_ELEMENTS_RO(om6->ospf6, node, ospf6)) {
+			ospf6_neighbor_show_common(vty, argc, argv, ospf6,
+						   idx_ipv4);
+		}
+		return CMD_SUCCESS;
+	}
+
+	ospf6 = ospf6_lookup_by_vrf_name(vrf_name);
+	OSPF6_CMD_CHECK_RUNNING(ospf6);
+	ospf6_neighbor_show_common(vty, argc, argv, ospf6, idx_ipv4);
```

Comment:
Same comment as before. This call for specific VRFs could also have been accommodated inside the for loop for 'all_vrf'.

Reply:
Resolved.